specification/network/resource-manager/readme.md

```diff
@@ -28,7 +28,7 @@ These are the global settings for the Network API.
 title: NetworkManagementClient
 description: Network Client
 openapi-type: arm
-tag: package-2021-03
+tag: package-2021-02-preview
```

Comment:
why do you need to change this tag?

Reply:
Because we need to this change to generate sdk PRs